router_test.go

```diff
@@ -471,6 +471,37 @@ func Test_Route_Static_HasPrefix(t *testing.T) {
 	body, err = io.ReadAll(resp.Body)
 	utils.AssertEqual(t, nil, err, "app.Test(req)")
 	utils.AssertEqual(t, true, strings.Contains(app.getString(body), "color"))
+
+	app = New()
+	app.Static("/css", dir)
+
+	resp, err = app.Test(httptest.NewRequest(MethodGet, "/css/style.css", nil))
+	utils.AssertEqual(t, nil, err, "app.Test(req)")
+	utils.AssertEqual(t, 200, resp.StatusCode, "Status code")
+
+	body, err = io.ReadAll(resp.Body)
+	utils.AssertEqual(t, nil, err, "app.Test(req)")
+	utils.AssertEqual(t, true, strings.Contains(app.getString(body), "color"))
+}
+
+func Test_Route_Static_SubApp(t *testing.T) {
+	t.Parallel()
+
+	dir := "./.github/testdata/fs/css"
```

Comment:
Either add a nolint or move it to a variable.

Reply:
Got. Thanks! I'll try it.